channeldb/db.go

```diff
@@ -286,6 +286,12 @@ var (
 			number:    31,
 			migration: migration31.DeleteLastPublishedTxTLB,
 		},
+		{
+			// Create a top level bucket which holds information
+			// about our node announcement.
+			number:    32,
+			migration: mig.CreateTLB(nodeAnnouncementBucket),
```

Comment:
you dont need a migration to add a new bucket.

Reply:
I removed it in another commit.